tests/NTest/NTestEnv.lua

```diff
@@ -0,0 +1,60 @@
+-- A series of convenient utility functions for use with NTest on NodeMCU
+-- devices under test (DUTs).
+
+local NTE = {}
+
+function NTE.hasC(m)
+  -- this isn't great, but it's what we've got
+  local mstr = node.info('build_config').modules
+  return mstr:match("^"..m..",") -- first module in the list
+      or mstr:match(","..m.."$") -- last module in the list
+      or mstr:match(","..m..",") -- somewhere else
+      or error("Missing C module " .. m)
```

Comment:
I would a has* function to return a boolean and not fail

Reply:
Fair.  If it returns a boolean the convention can be that tests `assert` on its result.  I think I was thinking they'd just call it, but that's pretty bogus.

Reply:
Its a question of naming. Call it checkFeat or even assertFeat and it will be ok to fail.
At $work we had a length discussion once what assure and ensure should do, we came up that one should just check and fail and the other should contain code to meet the expected requirement. But I can't remember the outcome.